images/base/entrypoint

```diff
@@ -54,27 +58,33 @@ fix_product_name() {
   # this is a small fix to hide the underlying hardware and fix issue #426
   # https://github.com/kubernetes-sigs/kind/issues/426
   if [[ -f /sys/class/dmi/id/product_name ]]; then
-    echo "kind" > /kind/product_name
+    echo 'INFO: faking /sys/class/dmi/id/product_name to be "kind"'
+    echo 'kind' > /kind/product_name
     mount -o ro,bind /kind/product_name /sys/class/dmi/id/product_name
   fi
 }
 
-link_kmsg() {
+fix_kmsg() {
   # In environments where /dev/kmsg is not available, the kubelet (1.15+) won't
   # start because it cannot open /dev/kmsg when starting the kmsgparser in the
   # OOM parser.
   # To support those environments, we link /dev/kmsg to /dev/console.
   # https://github.com/kubernetes-sigs/kind/issues/662
-  if [[ ! -e /dev/kmsg ]] && [[ -e /dev/console ]]; then
-    ln -s /dev/console /dev/kmsg
+  if [[ ! -e /dev/kmsg ]]; then
+    if [[ -e /dev/console ]]; then
+      echo 'WARN: /dev/kmsg does not exist, symlinking /dev/console' >&2
+      ln -s /dev/console /dev/kmsg
+    else
+      echo 'WARN: /dev/kmsg does not exist, nor does /dev/console!' >&2
+    fi
```

Comment:
I don't want lots of branching in this script, but I think branching slightly more to get better logs will help us later.